docs/team/daniel.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: Daniel's Project Portfolio Page
+---
+### Project: ContacX
+
+ContacX is for professionals who have many clients, current and prospective, who may possibly work with sensitive personal information and documents related to each of those clients. They have busy schedules with many appointments & meetings, and must be able to find available time slots in their schedules quickly.
+
+ContacX will allow for efficient management of a large list of contacts, allowing the mental workload of remembering people and related scheduled tasks to be offloaded. It will also provide an efficient way of efficiently organizing resources, while having minimal downtime with seamless transition from existing solutions.
+
+Given below are my contributions
+
+* **New Feature**: Import/Export
+    * What it does: `to be added soon`
+    * Justification:  `to be added soon`
+    * Highlights:  `to be added soon`
+    * Credits:  `to be added soon`
+
+* **Code contributed**: `to be added soon`
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the various `import` and `export` functions
+    * Developer's Guide:
+        * `to be added soon`
+* **Community**: `to be added soon`
```

Comment:
I would like to request the template retaining more of the original sections. See my PR #66.

Reply:
okay will follow thanks!